tools/experiments/experiments.js

```diff
@@ -424,6 +424,13 @@ const EXPERIMENTS = [
     spec: 'https://github.com/ampproject/amphtml/issues/20395',
     cleanupIssue: 'https://github.com/ampproject/amphtml/issues/20394',
   },
+  {
+    id: 'inabox-viewport-friendly',
```

Comment:
Since we want to run experiments on driven by the ad server, you'll need to add this to `allow-doc-opt-in` in `prod-config.json` and `canary-config.json`

Reply:
I see. Can this be done in the same PR? Since config changes are supposed to be separate.

Reply:
Not a problem to add it later.  It's just a thing you'll need to add before you can start running experiments.